conf/upgrade.yaml

```diff
@@ -0,0 +1,53 @@
+UPGRADE:
+  FROM_VERSION: "6.8"
+  TO_VERSION: "6.9"
+  OS: "rhel7"
+  DISTRO: ""
+  CAPSULE_AK:
+    CAPSULE_AK_RHEL6: "capsule_rhel6"
+    CAPSULE_AK_RHEL7: "capsule_rhel7"
+    CAPSULE_AK_RHEL8: "capsule_rhel8"
```

Comment:
is there any CAPSULE_AK_RHEL8 ?

Reply:
no, it is not supportable, but in the future, it could be that's why I keep it.